build_tools/generate_authors_table.py

```diff
@@ -75,23 +76,36 @@ def get_contributors():
     members |= {"Angel Soler Gollonet"}
     # remove CI bots
     members -= {"sklearn-ci", "sklearn-lgtm", "sklearn-wheels"}
-    triage_team -= core_devs  # remove ogrisel from triage_team
+    contributor_experience_team -= (
+        core_devs  # remove ogrisel from contributor_experience_team
+    )
 
-    emeritus = members - core_devs - triage_team
+    emeritus = members - core_devs - contributor_experience_team - comm_team
+    emeritus_comm_team = {"reshamas"}
```

Comment:
This one is hard to define programatically, I guess we have to hard code it

Reply:
Could we somehow use `doc/communication_team_emeritus.rst`?

Reply:
hehe this script is what generates ``communication_team_emeritus.rst``

Reply:
😊